pytest.ini

```diff
@@ -0,0 +1,3 @@
+[pytest]
+testpaths = tests
+addopts = -ra --tb=short
```

Comment:
is that necessary for this fix? if so why?

Reply:
It's not necessary for the fix. See the commit message of https://github.com/linkchecker/linkchecker/pull/319/commits/42c75b5ef9dc3e6f760d3c8964ff8dfdb68e676d to see why I did this (GitHub makes it annoyingly hard to see individual commit messages).

Reply:
ah, sorry, should have looked... makes sense, thanks!